db/seeds.rb

```diff
@@ -5,14 +5,15 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
-1.upto(5) {|i| Employee.create!(email: "test#{i}@test.com", password: 'password') }
-
-# 5.times do
-# Kudo.create(title: Faker::name, content: Faker::Address.street_address, giver_id: Employee.first.id, receiver_id: Employee.last.id)
-# end
+1.upto(5) {|i| Employee.create(email: "test#{i}@test.com", password: 'password') }
```

Comment:
Missing spaces before `| i|`. When using curly brackets a space should be inserted, like: hash = { foo: bar }